src/darwin/Framework/CHIPTests/MTRXPCListenerSampleTests.m

```diff
@@ -439,6 +439,19 @@ - (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSEr
     _expectation = nil;
 }
 
+- (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSError *)error deviceId:(NSNumber * _Nullable)deviceId
+{
+    XCTAssertEqual(error.code, 0);
+    [_expectation fulfill];
```

Comment:
This seems like a strange thing to do, if we are dispatching _both_ commissioningComplete notifications.

Reply:
remove it.